Dockerfile

```diff
@@ -1,20 +1,11 @@
-FROM ubuntu:20.04 as builder
-
-# Clone and build Blackbox
-RUN apt-get update && \
-    apt-get install -y build-essential git-core && \
-    git clone https://github.com/StackExchange/blackbox.git && \
-    cd blackbox && \
-    make copy-install
-
 FROM python:3.10-slim-bullseye
 LABEL maintainer "DataMade <info@datamade.us>"
 
 RUN apt-get update && \
     apt-get install -y libpq-dev gcc gdal-bin gnupg && \
     apt-get install -y libxml2-dev libxslt1-dev antiword unrtf poppler-utils \
                        tesseract-ocr flac ffmpeg lame libmad0 libsox-fmt-mp3 \
-                       sox libjpeg-dev swig libpulse-dev curl && \
+                       sox libjpeg-dev swig libpulse-dev curl git && \
```

Comment:
Need `git` to install `django-councilmatic` from the branch, will remove when new release is cut.